interface/clickmap/template/general_new.html

```diff
@@ -24,9 +24,11 @@
             </div>
         </div>
         <div class="col-12">
-            <button class="btn btn-primary btn-save" id="btn_save">{xl t="Save"}</button>
-            <button class="btn btn-secondary btn-delete" id="btn_clear">{xl t="Clear"}</button>
-            <button class="btn btn-secondary btn-cancel" id="cancel">{xl t="Cancel"}</button>
+            <div class="btn-group">
+                <button class="btn btn-primary btn-save" id="btn_save">{xl t="Save"}</button>
+                <button class="btn btn-secondary btn-delete" id="btn_clear">{xl t="Clear"}</button>
+                <button class="btn btn-secondary btn-cancel" onclick="top.restoreSession(); location.href='javascript:parent.closeTab(window.name, false)'">{xl t="Cancel"}</button>
```

Comment:
this doesn't look right

Reply:
disregard my comment above. you are using same cancel construct as other forms, so good